<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendDialogData</class>
 <widget class="QDialog" name="LegendDialogData">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>420</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Legend and Configuration for: </string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Graph </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox1">
        <item>
         <property name="text">
          <string>0 - Lines/Symbols</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 - Cumulative</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pTitle"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pAxis_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Axis Setup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="TextLabel1">
        <property name="text">
         <string>Abscissa grid</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="pAxisX">
        <property name="toolTip">
         <string>Set the maximum number of major scale intervals for  bottom axis</string>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QLineEdit" name="pXname">
        <property name="toolTip">
         <string>Change the title of a bottom axis</string>
        </property>
        <property name="maxLength">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="TextLabel1_3">
        <property name="text">
         <string>Ordinate grid</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="pAxisY">
        <property name="toolTip">
         <string>Set the maximum number of major scale intervals for  left axis</string>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QLineEdit" name="pYname">
        <property name="toolTip">
         <string>Change the title of a left axis</string>
        </property>
        <property name="maxLength">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel1_2">
        <property name="text">
         <string>Graph</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="TextLabel1_2_2">
        <property name="text">
         <string>x:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QLineEdit" name="pX0">
        <property name="toolTip">
         <string>Change minimum x of the scale</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="pXn">
        <property name="toolTip">
         <string>Change  maximum x of the scale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="TextLabel1_2_2_2">
        <property name="text">
         <string>y:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QLineEdit" name="pY0">
        <property name="toolTip">
         <string>Change minimum y of the scale</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="pYn">
        <property name="toolTip">
         <string>Change maximum y of the scale</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="TextLabel1_2_3">
        <property name="text">
         <string>Fragment</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="TextLabel1_2_2_3">
        <property name="text">
         <string>x:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QLineEdit" name="pfX0">
        <property name="toolTip">
         <string>Minimum</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="pfXn">
        <property name="toolTip">
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="TextLabel1_2_2_2_2">
        <property name="text">
         <string>y:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="3">
       <widget class="QLineEdit" name="pfY0">
        <property name="toolTip">
         <string>Minimum</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEdit" name="pfYn">
        <property name="toolTip">
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Label font:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QPushButton" name="pBtnChangeFont">
        <property name="text">
         <string>Change &amp;Font...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4" colspan="2">
       <widget class="QLineEdit" name="pLabelFont">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>This label font is common for all graphs</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QPushButton" name="buttomColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>C&amp;hange Color...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4" colspan="2">
       <widget class="QFrame" name="pColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pAxisX</tabstop>
  <tabstop>pXname</tabstop>
  <tabstop>pYname</tabstop>
  <tabstop>pX0</tabstop>
  <tabstop>pXn</tabstop>
  <tabstop>pY0</tabstop>
  <tabstop>pYn</tabstop>
  <tabstop>pfX0</tabstop>
  <tabstop>pfXn</tabstop>
  <tabstop>pfY0</tabstop>
  <tabstop>pfYn</tabstop>
  <tabstop>pBtnChangeFont</tabstop>
  <tabstop>pLabelFont</tabstop>
  <tabstop>comboBox1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
